notes.md

```diff
@@ -0,0 +1,127 @@
+# aic-kube MaaS Orchestration #
```

Comment:
Okay. I do want it in the repo to get feedback. I'll open issues for the major sections as they seem more efficient for collaboration.